setup/helm/varadhi/values.yaml

```diff
@@ -212,12 +219,12 @@ messaging:
   configMapName: messaging
   pulsar:
     adminOptions:
-      serviceHttpUrl: "http://192.168.1.3:8080"
+      serviceHttpUrl: "http://host.docker.internal:8080"
```

Comment:
may be time to decide the hierarchy of values files. Suggestion/TBD : values -> fcp.values ->  role specific i.e (controller|consumer|server|standalone) and override only required content in higher level files ?